R/html_dependencies.R

```diff
@@ -16,12 +16,7 @@ NULL
 #' @rdname html-dependencies
 #' @export
 html_dependency_jquery <- function()  {
-
-  htmlDependency(
-    name = "jquery",
-    version = "1.11.3",
-    src = pkg_file("rmd/h/jquery"),
-    script = "jquery.min.js")
+  jquerylib::jquery_core()
```

Comment:
What about already pinning version 3 to prevent any undesired future change if/when **jquerylib** changes its default ? 

````diff
-jquerylib::jquery_core()
+jquerylib::jquery_core(major_version = 3)
````

Reply:
I guess **jquerylib** won't change the version here suddenly (Carson will probably notify us), but I can definitely see the value of being a little conservative by pinning the version. To make it easier for ourselves to test future versions of jQuery, we can use an option to specify the version and default it to 3, e.g.,

```r
jquerylib::jquery_core(major_version = getOption('rmarkdown.jquery.version', 3))
```

Reply:
Oh I like the option ! I don't have this habits but this is great to have some config flag around for us to use.